modules/git/repo.go

```diff
@@ -152,7 +153,8 @@ func CloneWithArgs(ctx context.Context, from, to string, args []string, opts Clo
 	if len(opts.Branch) > 0 {
 		cmd.AddArguments("-b", opts.Branch)
 	}
-	cmd.AddArguments("--", from, to)
+	cmd.AddArguments("--", from, to).
+		SetDescription(fmt.Sprintf("clone from %s to %s", util.NewStringURLSanitizer(from, true).Replace(from), to))
```

Comment:
I think we have to leave this one to be autosanitization. This description lacks important information about the command.

```suggestion
	cmd.AddArguments("--", from, to)
```